Source/Core/Constants/Constants+Style.swift

```diff
@@ -13,6 +13,11 @@ extension Constants {
 
       static let animationLength: TimeInterval = 0.25
 
+        enum BorderStyle: Double {
+            case cornerRadius = 10
+            case borderWidth = 1
+        }
```

Comment:
Correct Constants use here would be:
```suggestion
        enum BorderStyle {
            static let cornerRadius: Double = 10
            static let borderWidth: Double = 1
        }
```

This will 
- enable the consumer to navigate object hierarchy in a logical manner
- prevent being able to have an initialiser
- remove need for `rawValue` at use site, offering a simpler and cleaner syntax